xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -200,3 +450,128 @@ int CDVDSubtitlesLibass::GetNrOfEvents() const
     return 0;
   return m_track->n_events;
 }
+
+int CDVDSubtitlesLibass::AddEvent(std::string text, double startTime, double stopTime)
```

Comment:
```suggestion
int CDVDSubtitlesLibass::AddEvent(std::string& text, double startTime, double stopTime)
```

Reply:
Please check if you really need to pass a `std:string` here as you'll endup feeding libass with a c_str

Reply:
done